tap.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for tap 14.4.2
+// Project: tap
+// Definitions by: Dmitry Mazurok <https://github.com/Eoksni>
```

Comment:
If this lives in the node-tap project, then it's going to be maintained by this project's maintainers.  I generally avoid personal attributions in files like this, because it sets up a weird expectation, and duplicates information already found in the git history.

If you'd prefer to have this work attached to your personal account for reputation or bragging rights, I totally understand.  In that case, it should be a separate project that you maintain.  Contributions to a project should be contributed to the project.

Reply:
No worries, I'll remove it. It is just a leftover from the template.

Reply:
Oh!  Ok, that makes total sense then.  (Same with the version number.)